File: Foundation/Scratch/Getting Started/c++/l002table.cpp
#include<iostream>
using namespace std;
void tableOfN(int n){
    for(int i = 1; i<=10; i++){
        cout<<to_string(n)+"X"+to_string(i)+"="+to_string(n*i)<<endl;
}   }
int main(){
    int of_n;    cin>>of_n;
    tableOfN(of_n);
    return 0;    }

File: Foundation/Scratch/Getting Started/java/l002table.java
import java.util.Scanner;
public class l002table{
    public static Scanner scn = new Scanner(System.in);
    public static void tableOfN(int n)
    {
        for(int i = 1; i<=10; i++)
        {
            System.out.println(n+"X"+i+"="+ n*i);
        }
    }
    public static void main(String[] args)
    {
        int of_n = scn.nextInt();
        tableOfN(of_n);
    }
}
